drivers/powerp-txt.c

```diff
@@ -34,13 +34,16 @@
 
 #include "powerp-txt.h"
 
-#define POWERPANEL_TEXT_VERSION "Powerpanel-Text 0.5"
+#include <ctype.h>
```

Comment:
Just in case: why `ctype.h`? Did the changes need some special integer type definition (like `size_t`, `uint16_t` etc.?)

If this is the case, then for portability's sake (and eventually formatting string preparation) better use our `nut_stdint.h` which tries to take care of platform specifics around that.

Reply:
No, that is for `isalpha()`. Got a warning on my first compile & grepped around to see several other files just including `ctype.h` directly when needed.